obtain_new_ids.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+"""
+Acquire a list of work IDs to be uploaded to Internet Archive.
+Purpose: automatic upload at regular intervals of newly-published works from selected publishers.
+Based on `iabulkupload/obtain_work_ids.py`.
+"""
+
+# Both third-party packages already included in thoth-dissemination/requirements.txt
+from internetarchive import search_items
+from thothlibrary import errors, ThothClient
+import json
+import logging
+from os import environ
+import sys
+
+logging.basicConfig(level=logging.INFO,
+                    format='%(levelname)s:%(asctime)s: %(message)s')
+thoth = ThothClient()
+
+# Check that a list of IDs of publishers whose works should be uploaded
+# has been provided as a JSON-formatted environment variable
+try:
+    publishers_env = json.loads(environ.get('ENV_PUBLISHERS'))
+except:
+    logging.error('Failed to retrieve publisher IDs from environment variable')
+    sys.exit(1)
+
+# Test that list is not empty - if so, the Thoth client call would erroneously
+# retrieve the full list of works from all publishers
+if len(publishers_env) < 1:
+    logging.error(
+        'No publisher IDs found in environment variable: list is empty')
+    sys.exit(1)
+
+# Test that all supplied publisher IDs are valid - if a mistyped ID was passed to the Thoth
+# client call, it would behave the same as a valid ID for which no relevant works exist
+for publisher in publishers_env:
+    try:
+        thoth.publisher(publisher)
+    except errors.ThothError:
+        # Don't include full error text as it's lengthy (contains full query/response)
+        logging.error('No record found for publisher {}: ID may be incorrect'.format(
+            publisher))
+        sys.exit(1)
+
+publishers = json.dumps(publishers_env)
+
+# Obtain all active (published) works listed in Thoth from the selected publishers.
+# `books` query includes Monographs, Edited Books, Textbooks and Journal Issues
+# but excludes Chapters and Book Sets. `bookIds` variant only retrieves their workIds.
+# thoth_works = thoth.bookIds(
```

Comment:
why is this commented out? I thought the idea was to use the query to replace lines 52-62